modules/core/02-client/keeper/keeper_test.go

```diff
@@ -252,20 +253,34 @@ func (suite KeeperTestSuite) TestGetAllGenesisClients() { //nolint:govet // this
 }
 
 func (suite KeeperTestSuite) TestGetAllGenesisMetadata() { //nolint:govet // this is a test, we are okay with copying locks
+	clientA, clientB := "07-tendermint-1", "clientB"
+
+	// create some starting state
+	suite.chainA.App.GetIBCKeeper().ClientKeeper.SetClientState(suite.chainA.GetContext(), clientA, &ibctm.ClientState{})
```

Comment:
took a brute force approach here, if someone wants to cleanup the tests, be my guest

Reply:
This helps test that the client and consensus state keys are not included in the exported metadata keys. It is weird, but 02-client genesis separates all these keys into separate buckets and iterates over them in separate instances

Reply:
That makes a lot of sense. Thank you! That would explain why that didn't surface for me earlier in the commit history when I stepped through the test with a debugger.
Perfect 🙌🏻